### PR TITLE
Set the table name in the MIN() MAX() query to work with models havin…

### PR DESCRIPTION
**Pull request by dougsbarbour**
…g eager loaded 1 to 1 associations.  Also, updated the tests and gem versions to be current.

### CODE REVIEW

**Comment by dougsbarbour** [comment]
I hope @haopingfan will merge my changes soon!

**Comment by khiav223577** [comment]
+1

File: Gemfile [hunk 1]
```diff
@@ -8,5 +8,5 @@ group :test do
   gem 'simplecov'
 end
 
-gemspec :path => "../"
+gemspec :path => "./"
```
Comment (haopingfan) [inline Gemfile]: Sorry for the late reply. What is this for?

Comment (dougsbarbour) [reply Gemfile]: That was not needed.  I removed it.

**Comment by haopingfan** [comment]
@dougsbarbour 
Thanks for your PR. Please resolve the conflict, then I will merge your PR.

**Comment by dougsbarbour** [comment]
Should be good to go now

**Comment by dougsbarbour** [comment]
Hi, could you release 0.3.2 on rubygems.org?

**Pull request merged**